fixture-monkey/src/test/java/com/navercorp/fixturemonkey/test/InnerSpecTest.java

```diff
@@ -827,4 +828,18 @@ void setNotNull() {
 
 		then(actual).isNotNull();
 	}
+
+	@Property
+	void keysForCollection() {
+		List<String> keyList = Arrays.asList("key1", "key2", "key3");
+		Map<String, String> actual = SUT.giveMeBuilder(MapObject.class)
```

Comment:
It would be nice to add a blank line in between so that the given-when-then pattern is more visible in the test.